polyglot/piranha/src/models/piranha_arguments.rs

```diff
@@ -12,140 +12,220 @@ Copyright (c) 2022 Uber Technologies, Inc.
 */
 
 use clap::Parser;
-use colored::Colorize;
+
+use super::{
+  default_configs::{
+    default_cleanup_comments, default_cleanup_comments_buffer,
+    default_delete_consecutive_new_lines, default_delete_file_if_empty, default_dry_run,
+    default_global_tag_prefix, default_input_substitutions, default_languages,
+    default_number_of_ancestors_in_parent_scope, default_path_to_code_base,
+    default_path_to_configurations, default_path_to_output_summaries, default_piranha_language,
+    default_substitutions,
+  },
+  language::PiranhaLanguage,
+};
+use crate::utilities::read_toml;
 use derive_builder::Builder;
 use getset::{CopyGetters, Getters};
-use log::info;
+use serde_derive::Deserialize;
 use std::{collections::HashMap, path::PathBuf};
-use tree_sitter::Language;
-
-use crate::{
-  config::CommandLineArguments,
-  models::piranha_config::PiranhaConfiguration,
-  utilities::{read_toml, tree_sitter_utilities::TreeSitterHelpers},
-};
 
-#[derive(Clone, Builder, Getters, CopyGetters, Debug)]
-#[builder(default)]
-/// Captures the processed Piranha arguments (Piranha-Configuration) parsed from `path_to_feature_flag_rules`.
+/// A refactoring tool that eliminates dead code related to stale feature flags.
+#[derive(Deserialize, Clone, Builder, Getters, CopyGetters, Debug, Parser, Default)]
+#[clap(name = "Piranha")]
 pub struct PiranhaArguments {
```

Comment:
We merge `CommandLineArgument` and `PiranhaConfiguration` into `PiranhaArguments`